bin/check-go-version

```diff
@@ -2,7 +2,7 @@
 
 set -eu -o pipefail
 
-VERSION="go1.14"
+VERSION="go1.15"
 
 GOLANG_VERSION=$(go version)
 if [[ $GOLANG_VERSION = *$VERSION* ]]; then
```

Comment:
We can update the install message below this too

Reply:
Whoops! Thanks for catching that!